mne/io/reference.py

```diff
@@ -155,6 +113,24 @@ def _apply_reference(inst, ref_from, ref_to=None):
         # non-CAR has been applied.
         if len(np.intersect1d(ref_to, eeg_idx)) > 0:
             inst.info['custom_ref_applied'] = FIFF.FIFFV_MNE_CUSTOM_REF_ON
+        # REST
+        if forward is not None:
```

Comment:
> It will be ignored

I see, I just found these lines that I had previously overlooked:
```python
    if ref_channels == 'REST':
        _validate_type(forward, Forward, 'forward when ref_channels="REST"')
    else:
        forward = None  # signal to _apply_reference not to do REST
```
Very good :)